.github/workflows/uffizzi-build.yml

```diff
@@ -16,30 +19,29 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v3
-        with:
-          path: backstage
 
       - name: setup-node
         uses: actions/setup-node@v3
         with:
           node-version: 16.x
           registry-url: https://registry.npmjs.org/
 
-      - name: backstage create-app | delete if already exists
+      - name: yarn install
+        uses: backstage/actions/yarn-install@v0.5.12
+        with:
+          cache-prefix: linux-v16
+
+      - name: Use Uffizzi's backstage app config
         run: |
-          rm -rf ./example-app
-          npx @backstage/create-app
-        env:
-          BACKSTAGE_APP_NAME: example-app
+          cp -f ./.github/uffizzi/uffizzi.production.app-config.yaml ./app-config.yaml
 
-      - name: yarn build
+      - name: typescript build
         run: |
-          yarn build:all
-        working-directory: ./example-app
+          yarn tsc:full
 
-      - name: Use Uffizzi's backstage app config
+      - name: backstage build
         run: |
-          cp ./backstage/.github/uffizzi/uffizzi.production.app-config.yaml ./example-app/app-config.production.yaml;
+          yarn build:all
```

Comment:
is this necessary? can we do like `yarn workspace backend build`, and rely on it actually building just the dependencies that it needs (including the app)?